### ISSUE TITLE
Text is not aligned center when print the bill

### ISSUE THREAD

**Issue by surjannegi** [DantSu/ESCPOS-ThermalPrinter-Android#537]
Hi All, I want to thanks @DantSu for the library . I am using the library to print the bill using Android app.

I am generating the print string using StringBuilder class in java. When I center align text using "[C]" , but it does not completely center align .

Please find the attachment.
![Untitled](https://github.com/user-attachments/assets/4c45eb9d-17bc-457d-9360-e908ea2a8b73)

Any help is appreciated

**Comment by harsh8088** [comment]
@surjannegi 
Try to change your printerWidthMM in EscPosPrinter


`            val printer = EscPosPrinter(UsbConnection(usbManager, usbDevice), 203, 80f, 32)
`

and I somehow managed to center the dynamic content.

Like this:

![Media (1)](https://github.com/user-attachments/assets/ceebf807-dd4e-4774-b200-9b256fddf7f2)

**Comment by surjannegi** [comment]
Thanks @harsh8088 for your response, will this align the complete text center align ? Because I want to center align the specific text , not the complete text.
I can see your screen shot , everything is center aligned ..

**Comment by harsh8088** [comment]
> Thanks @harsh8088 for your response, will this align the complete text center align ? Because I want to center align the specific text , not the complete text. I can see your screen shot , everything is center aligned ..

Yes, I have added multiline center text too.  Please have a look at my code here  [https://github.com/DantSu/ESCPOS-ThermalPrinter-Android/issues/538](url)

**Comment by surjannegi** [comment]
I clicked on the above link , i can't see any code there ..

**Comment by harsh8088** [comment]
> I clicked on the above link , i can't see any code there ..

Check the latest Open Issue(Font Issue):
https://github.com/DantSu/ESCPOS-ThermalPrinter-Android/issues/538

**Comment by surjannegi** [comment]
Thanks @harsh8088 , will check that

**Comment by Yiwei099** [comment]
@surjannegi
[DrawingSupport](https://github.com/Yiwei099/DrawingSupport) This library can meet your needs, and each individual text can have its own custom alignment；Just need to set TextParam.align
